packages/web-components/src/components/ic-checkbox/ic-checkbox.tsx

```diff
@@ -67,15 +67,20 @@ export class Checkbox {
   @Element() host: HTMLIcCheckboxElement;
 
   /**
-   * Emitted when a checkbox has been checked
+   * @deprecated This event should not be used anymore. Use icCheck instead.
    */
   @Event() checkboxChecked: EventEmitter<void>;
 
+  /**
+ * Emitted when a checkbox has been checked.
+ */
+  @Event() icCheck: EventEmitter<void>;
```

Comment:
Also, I'm wondering if we should call this 'icCheckboxCheck' if we're calling the event on the radio buttons 'icRadioCheck'? I guess ideally we would call them both 'icCheck' but I guess it's possible that may cause issues with event listeners if you have both radio buttons and checkboxes on a page and they're using the same event name

Reply:
I think maybe use icCheck for both. We already use icFocus, icBlur on multiple components. If you were creating an event listener, you would set it on the component, I would have thought?

Reply:
I tried using icCheck for both and it did seem to cause issues, however that might have been if I hadn't updated other values of it. I'll have another look :)